bindgen/ir/IR.cpp

```diff
@@ -353,3 +365,47 @@ IR::~IR() {
     variables.clear();
     varDefines.clear();
 }
+
+void IR::removeUnusedExternalTypes() {
```

Comment:
I don't think it makes sense to remove types, since we loose a lot of information. We need to external types to correctly reference types from other modules, like `stdio.FILE`. It would be better to update the generators to skip when `!isInMainFile` and then have them also prefix the module path when referencing an external type.